environment.py

```diff
@@ -901,5 +901,15 @@ def env() -> Mapping[str, Optional[str]]:
         # in the Azul VPC. This subnet can't overlap the VPC CIDR and the subnet
         # mask must be less than 22 bits.
         #
-        'azul_vpn_subnet': None
+        'azul_vpn_subnet': None,
+
+        # FIXME: Remove azul_proc_sys_crypto
+        #        https://github.com/DataBiosphere/azul/issues/6675
+        # When building the Azul dockerfile on a FIPS mode enabled system (e.g.
+        # GitLab), this variable should be set to `/proc/sys/crypto`, the path
+        # where a `fips_enabled` file will be mounted. This is required for the
+        # command `apt-get update` to succeed which would otherwise fail on
+        # Debian bookworm with FIPS mode enabled.
+        #
```

Comment:
Empty lines between paragraphs.

```suggestion
        # When building the Azul dockerfile on a FIPS mode enabled system (e.g.
        # GitLab), this variable should be set to `/proc/sys/crypto`, the path
        # where a `fips_enabled` file will be mounted. This is required for the
        # command `apt-get update` to succeed which would otherwise fail on
        # Debian bookworm with FIPS mode enabled.
        #
        # FIXME: Remove azul_proc_sys_crypto
        #        https://github.com/DataBiosphere/azul/issues/6675
        #
```